docs/src/main/sphinx/release/release-365.md

```diff
@@ -0,0 +1,126 @@
+# Release 365 (XX Dec 2021)
+
+## General
+
+* Add support for [`TRUNCATE TABLE`](/sql/truncate). ({issue}`8921`)
+* Add support for time travel queries. ({issue}`8773`)
+* Add [`contains`](ip_address_contains) function to check whether a CIDR contains an IP address. ({issue}`9654`)
+* Report connector metrics in `EXPLAIN ANALYZE VERBOSE`. ({issue}`9858`)
+* Report operator input distribution in ``EXPLAIN ANALYZE VERBOSE``. ({issue}`10133`)
```

Comment:
input size distribution

Reply:
Size or row count? cc @sopel39

Reply:
it's about positions count, not size.
Maybe: ```Report operator input positions distribution ...`?